### PR TITLE
adding time stamp for entire project and step

### PR DESCRIPTION
**Pull request by Galileo1**
added time taken by each step 
added time taken by entire project 
changed the scenario panel heading to remove overlapping
image

### CODE REVIEW

File: lib/reporter.js [hunk 1]
```diff
@@ -163,10 +173,23 @@ var generateReport = function(options) {
...
+            if (feature.time){
+                suite.totalTime +=  feature.time
+            }
+
             return suite;
 
         });
 
+        //total suite/project runtime
+        if (suite.totalTime / 60000000000 >= 1) {
```
Comment (gkushang) [inline lib/reporter.js]: Can you define a variable for the magic number `60000000000` value?

**Comment by gkushang** [comment]
@Galileo1 I like this feature and will be pretty useful for the users. Thanks for implementing it! I will merge it once review is done. Sorry for delaying the resonse.

**Comment by gkushang** [comment]
Can you fix/undo the alignment of `label-container` for scenarios _success/failure_ numbers? It should be right-aligned.

screen shot 2016-09-16 at 12 16 13 pm

**Comment by thyagab** [comment]
@Galileo1 @gkushang This is a cool feature. Will this provide the total elapsed time  when we give jsondir as well?

File: lib/reporter.js (hunk 1 above)
Comment (Galileo1) [inline lib/reporter.js]: sure..

**Comment by Galileo1** [comment]
@gkushang reg label-container - my css skills sucks but I will make it work

**Comment by gkushang** [comment]
@thyagab that's a good point. We should definitely test it out with `jsonDir`

**Comment by gkushang** [comment]
@Galileo1 I'm looking into fixing CSS so that we can merge this feature

**Comment by gkushang** [comment]
@Galileo1 merging this and will fix CSS. Lets discuss on how to display total duration here -  https://github.com/gkushang/cucumber-html-reporter/issues/26 and enable it later, but lets publish what you have so far. Sounds good?

**Pull request merged**